src/Integration/Notifications/VisualStudioStatusBarHelper.cs

```diff
@@ -39,6 +40,8 @@ public static void RemoveStatusBarIcon(FrameworkElement statusBarIcon)
 
         public static void AddStatusBarIcon(FrameworkElement statusBarIcon)
         {
+            Debug.Assert(statusBarIcon.Parent == null, "Icon should not already have been sited");
```

Comment:
The bug was showing up as a WPF error; the solution tracker was raising too many events, causing this code to be try to add the same element to the visual tree multiple times.